tests/Doctrine/Tests/CouchDB/Functional/BulkUpdaterTest.php

```diff
@@ -43,47 +43,6 @@ public function testExecute()
         $this->assertEquals(array(), $response->body);
     }
 
-    /**
-     * @depends testExecute
-     */
-    public function testSetAllOrNothing()
```

Comment:
Why has this been removed.

Reply:
all_or_nothing is no longer supported by the bulk_docs API 
http://docs.couchdb.org/en/2.0.0/whatsnew/2.0.html#id3